data/resources.json

```diff
@@ -35,7 +35,7 @@
   "aboutOrganizers": "<a href='http://lviv.gdg.org.ua' target='_blank'>GDG Lviv</a> with support of 10 GDGs from all around Ukraine is proud organizer of DevFest Ukraine",
   "openTeamPage": "Open team page",
   "whatIsGdg": "What is GDG?",
-  "aboutGdg": "Google Developers Group (GDG) - is open and volunteer geek community who create exciting projects and share experience about Google technologies with a passion",
+  "aboutGdg": "Google Developers Group (GDG) - is open community of volunteers who come up with creating and exciting ideas with the aim of coming up with solutions to day to day problems within the society.(both locally and globally)",
```

Comment:
Is this creative or creating? 🤔

Reply:
it should be creative and the beginning part should be is an open not  is open...let me fix that